### PR TITLE
Copy cert to system only if both AdGuard certs exist in user storage

### PR DESCRIPTION
**Pull request by grumaxxx**

### CODE REVIEW

File: module/post-fs-data.sh [hunk 1]
```diff
@@ -16,8 +16,15 @@ MODDIR=${0%/*}
 #    They get there if a certificate is "disabled" in the security settings.
 #    Apps will reject certs that are in the `cacerts-removed`.
 AG_CERT_HASH=0f4ed297
+AG_INTERMEDIATE_CERT_HASH=47ec1af8
 AG_CERT_FILE=$(ls /data/misc/user/*/cacerts-added/${AG_CERT_HASH}.* | sort | tail -n1)
-mv -f ${AG_CERT_FILE} ${MODDIR}/system/etc/security/cacerts/${AG_CERT_HASH}.0
+AG_INTERMEDIATE_CERT_FILE=$(ls /data/misc/user/*/cacerts-added/${AG_INTERMEDIATE_CERT_HASH}.* | sort | tail -n1)
+
+if [ ! -e "${AG_INTERMEDIATE_CERT_FILE}" ]; then
```
Comment (sfionov) [inline module/post-fs-data.sh:23]: May be check both?

File: module/post-fs-data.sh [hunk 2]
```diff
@@ -16,8 +16,15 @@ MODDIR=${0%/*}
...
+    exit 1
```
Comment (sfionov) [inline module/post-fs-data.sh:24]: When module fails, how it is displayed in Magisk?

Comment (grumaxxx) [reply module/post-fs-data.sh:24]: it displays as successfully installed

Comment (sfionov) [reply module/post-fs-data.sh:24]: Even after reboot?

Comment (grumaxxx) [reply module/post-fs-data.sh:24]: Yeah, I double-checked

File: module/post-fs-data.sh (hunk 1 above)
Comment (sfionov) [reply module/post-fs-data.sh:23]: At least we will not do extra work such as removing certs-removed and configuring selinux

**Review by sfionov — APPROVED** [review]

**Pull request merged**